cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -55,233 +62,240 @@ const F_RETAIN_REDUNDANT_REX: u32 = 1;
 /// indicating a 64-bit operation.
 const F_CLEAR_REX_W: u32 = 2;
 
-/// For specifying the legacy prefixes (or `PfxNone` if no prefix required) to
+/// For specifying the legacy prefixes (or `None` if no prefix required) to
 /// be used at the start an instruction. A select prefix may be required for
```

Comment:
I'm not sure what the sentence "A select prefix .. " means (what's a "select" prefix?)  Maybe rm this sentence?  In any case none of this machinery will work for creating VEX- or EVEX-prefixed instructions; we'll have to redo it for those.

Reply:
This is preexisting :-) Maybe "a given prefix" would make more sense here?